pkgs/top-level/release-cross.nix

```diff
@@ -89,101 +89,26 @@ in
     guile = nativePlatforms;
   };
 
-  darwinToAarch64 = let
-    crossSystem = {
-      config = "aarch64-apple-darwin14";
-      arch = "arm64";
-      libc = "libSystem";
-    };
-  in mapTestOnCross crossSystem darwinCommon;
+  darwinToAarch64 = mapTestOnCross lib.systems.examples.iphone64 darwinCommon;
 
-  darwinToArm = let
-    crossSystem = {
-      config = "arm-apple-darwin10";
-      arch = "armv7-a";
-      libc = "libSystem";
-    };
-  in mapTestOnCross crossSystem darwinCommon;
+  darwinToArm = mapTestOnCross lib.systems.examples.iphone32 darwinCommon;
```

Comment:
It's just "darwinToArm" is a little weird compared to the others below. I guess if this is impure you're trying to say it only works on Darwin systems? I guess you're just keeping with the old naming so should be fine.